pkg/acs/lib/acsrej/rej_command.c

```diff
@@ -21,238 +23,389 @@ static int getArgT (char **, int, int *, char *);
    ----------------
    27 Aug 98  -  Adapted from CALSTIS2 cs2_command.c (WJ Hack)
    27 Jul 15  -  Dynamic size for input filenames (PL Lim)
+   04 Apr 25  -  Replace argument parser with argp
 */
 
-int rej_command (int argc, char **argv, char **input, char *output,
-                clpar *par, int newpar[])
-{
+extern int status;
+
+// Arguments that exit early
+#define OPT_GITINFO 1000
```

Comment:
Where did you get the values for these? Are these new codes? Should they be in a header file?

Reply:
The field where they are used can be an ascii character (short option) or an arbitrary value that argp uses to map a long option back to "something". The "something" is the number.